BookingsApi/BookingsApi.DAL/Helper/CloneHearingToCommandMapper.cs

```diff
@@ -52,10 +52,10 @@ public static class CloneHearingToCommandMapper
 
             var duration = 480;
             var command = new CreateVideoHearingCommand(new CreateVideoHearingRequiredDto(
-                    hearing.CaseType, hearing.HearingType, newDate, duration, hearing.HearingVenue, cases),
```

Comment:
just wondering if we should leave this intact, so that cloned V1 bookings retain their hearing type (as they presumably will still be created with hearing types)

Reply:
It still exists, i've just moved it into the option DTO on line 58

Reply:
ah I see, makes sense